web/modules/custom/joinup_video/joinup_video.module

```diff
@@ -25,3 +29,35 @@ function joinup_video_form_video_embed_dialog_alter(&$form, FormStateInterface $
     . '<br />EC url example: http://europa.eu/123abc!123'
     . '<br />EC video example: https://ec.europa.eu/avservices/video/player.cfm?sitelang=en&ref=ABC12345';
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function joinup_video_preprocess_video_embed_iframe(&$variables) {
+  // Override the default url and pass it to the ec cck url but ignore iframes
+  // that refer to the same site.
+  if (!empty($variables['url']) && !UrlHelper::externalIsLocal($variables['url'], \Drupal::request()->getSchemeAndHttpHost())) {
+    // video_embed_field prints every part of the URL separately and prints some
+    // key characters in the twig directly (e.g. the '#' for the anchor).
+    // This makes it difficult to properly encode the full URL to pass it to the
+    // cck domain. Pre-create the URL and pass it as an encoded URL.
+    // The query parameters are being built and constructed using the
+    // http_build_query rather than allowing URL to parse them. This is because
+    // the \Drupal\Component\Utility\UrlHelper::buildQuery constructs the query
+    // parameters differently than how PHP does and the result is that boolean
+    // values are not parsed properly.
+    // @todo: Remove the workaround once this is fixed in core.
+    // @see \Drupal\Component\Utility\UrlHelper::buildQuery
+    // @see https://www.drupal.org/project/drupal/issues/2248257
+    $query_parameters = http_build_query($variables['query']);
+    $url = Url::fromUri(rtrim($variables['url']) . '?' . $query_parameters, [
```

Comment:
Probably a better idea is to prepare first `$variables['query']` and replace booleans with strings. I.e. `TRUE` -> `'1'`, `FALSE` -> `'0'`. Then we can use `Url::fromUri()` in a standard way.